flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/sink/PulsarSinkBuilder.java

```diff
@@ -126,9 +123,11 @@ public class PulsarSinkBuilder<IN> {
      *
      * @param adminUrl The url for the PulsarAdmin.
      * @return this PulsarSinkBuilder.
+     * @deprecated this method will return builder directly
```

Comment:
Perhaps add a link to this PR as background. But it's OK that we do it in a follow-up patch.